src/main/msp/msp_protocol.h

```diff
@@ -62,7 +62,7 @@
 #define MSP_PROTOCOL_VERSION                0
 
 #define API_VERSION_MAJOR                   1  // increment when major changes are made
-#define API_VERSION_MINOR                   44 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
+#define API_VERSION_MINOR                   45 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
```

Comment:
Don't change this. The API version will be updated administratively after 4.3 is released.

Reply:
Thanks! I have reverted the API version change in both this MR and [the configurator one](https://github.com/betaflight/betaflight-configurator/pull/2798) (where I left the minimum version assertions but I pointed them to current version; these should be straightforward to update after the 4.3 release).